test/unittests/parser_test.cpp

```diff
@@ -257,6 +257,48 @@ TEST(parser, function_section_with_multiple_functions)
     EXPECT_EQ(module.funcsec[3], 0xff);
 }
 
+TEST(parser, table_single_min_limit)
+{
+    const auto section_contents =
+        bytes{} + uint8_t{0x01} + uint8_t{0x70} + uint8_t{0x00} + uint8_t{0x7f};
```

Comment:
The file uses `uint8_t{}` allover the place, I suggest it is fixed in a follow up PR. Currently these tests mirror the memory section tests (as they are very similar).

Reply:
In #80.